homeassistant/components/reisingerbridge/strings.json

```
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "device_name": "Device name",
+          "device_key": "Device key",
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
```

This doesn't seem to be part of the code?

```suggestion
          "port": "[%key:common::config_flow::data::port%]"
```